spdx-utils/src/test/kotlin/SpdxExpressionTest.kt

```diff
@@ -367,6 +368,118 @@ class SpdxExpressionTest : WordSpec() {
             }
         }
 
+        "applyChoice()" should {
+            "return the choice for a simple expression" {
+                val expression = "a".toSpdx()
+                val choice = "a".toSpdx()
+
+                val result = expression.applyChoice(choice)
+
+                result shouldBe "a".toSpdx()
+            }
+
+            "throw an exception if the user chose a wrong license for a simple expression" {
+                val expression = "a".toSpdx()
+                val choice = "b".toSpdx()
+
+                shouldThrow<InvalidLicenseChoiceException> { expression.applyChoice(choice) }
+            }
+
+            "return the choice if the subExpression contains multiple choices" {
+                val expression = "a OR b OR c".toSpdx()
+                val choice = "a".toSpdx()
+
+                val result = expression.applyChoice(choice)
```

Comment:
Currently this is the same test as "apply the choice if the expression contains multiple choices" below, according to the test name I think you wanted to pass a subexpression here.

Reply:
It was a leftover from before the API change where not more than one choice were allowed. As now choices can be applied I removed this test.